src/MetaImageFormat.jl

```diff
@@ -501,7 +506,7 @@ function parse_header(io)
     # between the header and data
     line = strip(readline(io))
     while !isempty(line)
-        idx = findfirst(line, '=')
+        idx = findfirst("=", line)[1]
```

Comment:
In a performance-sensitive situation, `findfirst(isequal('='), str)` would presumably be faster, but this is not performance-sensitive so either is fine.